CHANGES

```diff
@@ -1,4 +1,15 @@
 0.1.0 (2017-06-14)
-======================
+==================
 
-- Initial release
\ No newline at end of file
+- Initial release
+
+0.1.1 (2017-06-14)
+==================
+
+- convert README to ReST
+
+0.2.0 (2017-06-??)
```

Comment:
Usually releases are listed in reverse chronological order. So latest release on top.